### PR TITLE
[massive update] add new feature and bug patches

### LINKED ISSUE #26: Feature Request: Add override settings for orientation information
It is possible for an operator to set orientation (and subject type, see #25 ) incorrectly such that conversion does not work properly. I'd like to request options to override the detected orientation to allow conversion to work properly.

### PR DESCRIPTION
**Pull request by dvm-shlee**
Fixes #26, #27, #68 etc [massive updates]

Changes proposed in this pull request:
- cleaned up old commented code for loader.py 509a2cf301ef5ac7221e1b77cdf90e7d3205ae0c and brkraw.py ec279a238613896cab52ed211489e69587a5c751
- override function added to loader.py 23da9566a9e4f369781e28e49ca459cb4caff225 and conducted to the brkraw.py 37ca96bc8ea31ec053f71f980650ca6a7a3a8de7, 87eafb08d5f185e8819ad2f73e0cdca9400f9abf
- a patch applied to the `tonii` to check input file datatype f23adb7bc08f4962bb701ed2fe0cd6e2475583a6
- bug fixed on `tonii_all` that not abled to load dataset (#27)  5c6d62fcd533c6752e103c826de20b602b466546
- added function to check localizer scan (based on the name of protocol, Localizer or TriPilot; case none-sensative) 90efec82bc273546e5ee3f5f15c4f4ad34c9963a
- add ignore localization option for `tonii` and `tonii_all` 7cd03fda62b586d0305c4d9bc0378a93e4362826
- a patch applied to the `bids_convert` to use is_localizer function instead of skipping the scan has 3 slice-packages a7528b4f50b3df64083a6ee9eb793321fa81b16d
- minor beautification of arg_parsers document for `tonii` 1d060496cb60a3b4fdda56845d68099a7ec38995
- nifti conversion now have correct qform and sform code as 1 and 0 respectively which solve Issue #68

@BrkRaw/Bruker

### CODE REVIEW

**Comment by gdevenyi** [comment]
I'm digging through the pile commits. This all looks great.

We don't need to do this now, but next time, reviews of changes is much easier if each change is a seperate PR.

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 setup(name='bruker',
       version=__version__,
       description='Bruker PvDataset Loader',
-      python_requires='>3.5, <3.8',
+      python_requires='>3.5, <3.9',
```
Comment (gdevenyi) [inline setup.py:22]: if you require >3.6, you could convert all your string formatting to f-strings

File: brkraw/scripts/brkraw.py [hunk 2]
```diff
@@ -683,5 +707,36 @@ def completeFieldsCreateFolders (df, filtered_dset, dset, multi_session, root_pa
...
+        return False
+
+
+def override_header(pvobj, subjtype, position):
+    """override subject position and subject type"""
+    import warnings
+    if position is not None:
+        try:
+            pvobj.override_position(position)
+        except:
```
Comment (gdevenyi) [inline brkraw/scripts/brkraw.py:726]: I think I would instead error out with incorrect.

File: brkraw/scripts/brkraw.py [hunk 3]
```diff
@@ -683,5 +707,36 @@ def completeFieldsCreateFolders (df, filtered_dset, dset, multi_session, root_pa
...
+            msg = "Incorrect position string [{}], using default position instead. Please check your input option.".format(position) + \
+                  "The position variable can be defiend as <BodyPart>_<Side>," + \
+                  "available BodyParts are (Head, Foot, Tail) and sides are (Supine, Prone, Left, Right). (e.g. Head_Supine)"
+            warnings.warn(msg)
+    if subjtype is not None:
+        try:
+            pvobj.override_subjtype(subjtype)
+        except:
+            msg = "Incorrect subject type [{}], using default subject type instead. Please check your input option.".format(subjtype) + \
```
Comment (gdevenyi) [inline brkraw/scripts/brkraw.py:735]: I'd rather error out

**Review by gdevenyi — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (dvm-shlee) [reply setup.py:22]: Might consider your suggested change (make it >3.6) to put items for the next version. Thank you for the suggestion!

File: brkraw/scripts/brkraw.py (hunk 2 above)
Comment (dvm-shlee) [reply brkraw/scripts/brkraw.py:726]: I put this feature for some service requests that do not provide a `subject` file.  (only provide the folder with a separate subject datasheet.
But agree that this needs to be more restricted and return error is the proper way. I will put this item for the next update.

File: brkraw/scripts/brkraw.py (hunk 3 above)
Comment (dvm-shlee) [reply brkraw/scripts/brkraw.py:735]: Same as above.

**Pull request merged**